src/wiseagents/core.py

```diff
@@ -665,45 +952,133 @@ class WiseAgentRegistry:
     contexts : dict[str, WiseAgentContext] = {}
     tools: dict[str, WiseAgentTool] = {}
     
+    config: dict[str, Any] = {}
+    
+    redis_db : redis.Redis = None
+    
+    @classmethod
+    def find_file(cls, file_name, config_directory="") -> str:
+        """
+        Find the file in the current directory or the home directory.
+        """
+        # Step 1: Check the current directory
+        if os.path.isfile(file_name):
+            return file_name
+        
+        # Step 2: Check the home directory
+        home_dir = os.path.expanduser("~")
+        home_path = os.path.join(home_dir, config_directory,file_name)
+        if os.path.isfile(home_path):
+            return home_path
+        
+        # If the file is not found in any of these locations, return None
+        raise FileNotFoundError(f"File '{file_name}' not found in current directory, home directory, as '{config_directory}'/{file_name}.")
```

Comment:
The comment here doesn't match what happens, i.e., None isn't returned in this case.